lib/myfinance/resources/payable_account.rb

```diff
@@ -26,7 +26,8 @@ def endpoints(id, entity_id, key)
         {
           create: '/entities/:entity_id/payable_accounts',
           pay: '/entities/:entity_id/payable_accounts/:id/pay',
-          undo_payment: '/entities/:entity_id/payable_accounts/:id/undo_payment'
+          undo_payment: '/entities/:entity_id/payable_accounts/:id/undo_payment',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.